pennylane/ops/qutrit/observables.py

```diff
@@ -14,13 +14,26 @@
 """
 This submodule contains the qutrit quantum observables.
 """
-from pennylane import numpy as np
+import numpy as np
```

Comment:
Why was this changed?

Reply:
This is because PL numpy wraps arrays inside tensor objects, so if someone calls the `compute_matrix` method, it would be wrapped as a tensor with `requires_grad=True`. This doesn't really break anything, but since this is meant to be a non-parametrized observable, it's matrix shouldn't be differentiable, and any PL code using the observable, such as the `DefaultQutrit` measurement functions, would cast the matrix to match the dtype of the device, so using PL numpy is really unnecessary.

Reply:
Ah good point! Thanks for clarifying. Yeah I see now in the qubit non-parametric ops and observable file this is done as well.